ui/widgets/nuxeo-select.js

```diff
@@ -197,6 +199,31 @@ import { IronResizableBehavior } from '@polymer/iron-resizable-behavior/iron-res
           value: null,
         },
 
+        /**
+         * The orientation against which to align the menu dropdown
+         * horizontally relative to the dropdown trigger.
```

Comment:
```suggestion
         * The orientation against which to align the menu dropdown horizontally relative to the dropdown trigger.
```
Right? Isn't the max `120` chars? Same below.

Reply:
TBH, I've just copied from `paper-dropdown-menu` ([here](https://github.com/PolymerElements/paper-dropdown-menu/blob/master/paper-dropdown-menu.js#L195-L205)) 😅 

Lint passed, so I guess it shouldn't be an issue 🤷🏼‍♂️